src/modality-specific-files/task-events.md

```diff
@@ -309,3 +309,95 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
     "VisionCorrection": "lenses"
 }
 ```
+
+### Continuously-sampled, stimulus-related signals
+
+!!! example "Example datasets"
+
+      At the time of writing, there are no
+      [example datasets](https://bids-standard.github.io/bids-examples/#dataset-index)
+      showcasing stimulus-related signals.
+      However, the following datasets shared on [OpenNeuro](https://openneuro.org)
+      include these type of signals and may be used (with caution, because they are not
+      "official" examples) as a reference when curating a new dataset:
+
+      -   [*Trial timing for multivariate pattern
+          analysis*](https://openneuro.org/datasets/ds000238/versions/00002)
+
+Signals related to stimuli (such as parameters of a film or audio stimuli) that are
+evenly recorded at a constant sampling frequency SHOULD be stored as follows.
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    <datatype>/
+        <matches>[_recording-<label>]_stim.tsv.gz
+        <matches>[_recording-<label>]_stim.json
+```
+
+For the template directory name, `<datatype>` can correspond to any data
+recording modality.
+
+In the template filenames, the `<matches>` part corresponds to task filename
+before the suffix.
+For example for the file `sub-control01_task-nback_run-1_bold.nii.gz`,
+`<matches>` would correspond to `sub-control01_task-nback_run-1`.
+
+!!! warning "Caution"
+
+    `<matches>[_recording-<label>]_stim.tsv.gz` files MUST NOT include a header line,
+    as established by the [common-principles](../common-principles.md#compressed-tabular-files).
+    As a result, when supplying a `<matches>[_recording-<label>]_stim.tsv.gz` file,
+    an accompanying `<matches>[_recording-<label>]_stim.json` MUST be present to indicate
+    the column names.
+
+The [`recording-<label>`](../appendices/entities.md#recording)
+entity MAY be used to distinguish between several recording files,
+as prescribed by the specifications for
+[physiological recordings](physiological-recordings.md).
+
+If the same continuous recording has been used for all subjects (for example in
+the case where they all watched the same movie), one file placed in the
+root directory (for example, `<root>/task-movie_stim.<tsv.gz|json>`) MAY be used
+and will apply to all `<matches>_task-movie_<matches>_<suffix>.<ext>` files.
+In the following example, the two `task-nback_stim.<json|tsv.gz>` apply
+to all the `task-nback` runs across the two available subjects:
```

Comment:
@effigies - I think your comment (https://github.com/bids-standard/bids-specification/pull/1750/files#r1541231904) can be transferred here.

The question would be whether this (already existing before this PR) text for stim files covers what you are indicating above (https://github.com/bids-standard/bids-specification/pull/1750/files#r1541231904).

The other facet of this problem (involving physio files) can be addressed then separately within #1753).

I'm resolving the old conversation to discuss anything relating stim under this comment or relating to physio with #1753.

Reply:
Since this is only a "transference" PR, I'll let it be like it was and open a sibling issue to #1753. I've realized that stim files present an additional challenge - you definitely can run the same stimuli (hence single stim file) in two different experiments (e.g., fMRI and EEG) performed separately in different moments.